src/pages/home/index.js

```diff
@@ -1,12 +1,355 @@
 import React from 'react'
 
-const Home = () => {
+const Home = (props) => {
+
+  const handleClick = () => {
+    props.history.push('/checkout')
+  }
+
+
+
   return(
-      <div>
-        <p>This is the start of you first real react project.</p>
-        <p>I've done some of the basic setup for you, like adding react-router for routing.</p>
-        <p>You should have a trello board with all the tasks you need to complete on it. Along with a design file from figma that you should work from.</p>
-        <p>This is going to be a much larger chalenge than any of your previous work, so make sure that you plan each part out and refer to notes!</p>
+      <div style= {{
+        backgroundColor:"grey", textAlign:"center", height: 1750, alignItems:"center", justifyContent:"center", display:"flex", flexDirection:"column"
+    }}>
+        <h3 style={{justifyContent:"flex-start", alignItems:"center", flex:1}}>Welcome to the store!</h3>
+        <button onClick={handleClick} style={{
+          backgroundColor:"gold",
```

Comment:
There's a lot of inline styles here and it's getting harder to read.
I think we should separate these into a css file